tests/test_setup.py

```diff
@@ -31,6 +32,20 @@ def get_args(yaml_path: str = '', cwl_inline_subworkflows: bool = False) -> argp
     return args
 
 
+def get_appended_args(yaml_path: str = '', suppliedargs: List[str] = []) -> argparse.Namespace:
```

Comment:
We should not have both get_args and get_appended_args. We should keep the body of get_appended_args, name it get_args, and update all of the call sites where cwl_inline_subworkflows = True.